hometask-11

Darya-Dubovik opened the pull request:
Написала код тренажера "слепой печати"

Darya-Dubovik committed:
hometask-11

Написала код тренажера "слепой печати"

mdubik777 commented on script.js:
```diff
@@ -0,0 +1,95 @@
+            wrongCount.textContent = ++wrongCount.textContent;
+        };
+        
+        isNewWord();
```
Вот тут, Даша,  кое-чего не хватает
![Screenshot_186](https://github.com/user-attachments/assets/67d0e000-58bb-4d4b-ab7c-008c3f99544b)
Игра уже закончилась,  но последняя буква не окрашена,  и в счётчике 4, а не 5. Это потому, что проверка на конец игры отрабатывет раньше,  чем isSuccess();. Чтобы это поправить ,  нужно эту функцию обернуть в setTimeout , время поставить 0,  и тогда она отработает чуть позже,  и всё будет правильно

mdubik777 commented on script.js:
```diff
@@ -0,0 +1,95 @@
+
+function checkNumber() {
+    if (correctCount.textContent >= 5) {
+        alert('Вы победили!');
```
Вот эти 4 строки и там и в след функции почти одинаковые. Вынеси их в отдельную функцию,  которая принимает текст алерта, и вызови её в двух местах

Darya-Dubovik committed:
hometask-11

Исправила код по комментарию наставника

mdubik777 commented on script.js:
```diff
@@ -0,0 +1,94 @@
+    currentWord = words[getRandomIndex()];
+    renderWord(currentWord);
+    index=0;
+    error=false;
```
Даша,  и ещё один момент)  с каждым новым словом счётчик количества ошибок именно в слове должен обнуляться. У тебя этого не происходит)

Darya-Dubovik committed:
hometask-11

Исправила ошибки в коде по комментарию наставника (обнуление счетчика ошибок в слове при переходе на следующее)
